blocks/algorithms/__init__.py

```diff
@@ -369,10 +369,25 @@ class Momentum(CompositeRule):
     momentum : float, optional
         The momentum coefficient. Defaults to 0.
 
+    Attributes
+    ----------
+    learning_rate : :class:`~tensor.SharedVariable`
+        A variable for learning rate.
+    momentum : :class:`~tensor.SharedVariable`
+        A variable for momentum.
+
+    See Also
+    --------
+    :class:`SharedVariableModifier` for a parameter decay during the
+    training.
+
     """
     def __init__(self, learning_rate=1.0, momentum=0.):
-        self.components = [Scale(learning_rate=learning_rate),
-                           BasicMomentum(momentum=momentum)]
+        scale = Scale(learning_rate=learning_rate)
+        momentum_alg = BasicMomentum(momentum=momentum)
```

Comment:
Please rename `momentum_alg` to `basic_momentum`. Sorry for being that picky, but we allow only a few very popular abbreviations in blocks, such as `init`, `param`, `var` and so on. `alg` does not belong to this category.

I will merge right after you fix that, honestly!